pkg/policy/consumer.go

```diff
@@ -93,6 +93,11 @@ func NewConsumable(id NumericIdentity, lbls *Identity, cache *ConsumableCache) *
 	return consumable
 }
 
+func (c *Consumable) GetIdentityFromConsumableCache(id NumericIdentity) *Identity {
```

Comment:
Can we rename this to `ResolveIdentity`?

Reply:
sure. Although I think I was following the Get* convention of ProxySource interface

Reply:
This one is from ConsumableCache. its not same as GetIdentityFromConsumable.
ResolveIdentityFromCache?

Reply:
done